Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 
 Compiler Features:
  * Control Flow Graph: Warn about unreachable code.
+ * Tests: Add infrastructure to ``isoltest`` for running semantic tests.
```

Comment:
I think it would be better to move that to "build system" or rename "Build system" to "infrastructure" or something like that. At least it is nothing that concerns the user.